ProjWidget.py

```diff
@@ -0,0 +1,151 @@
+from PyQt5 import QtCore
+from PyQt5.QtGui import *
+from PyQt5.QtWidgets import *
+from PyQt5.QtCore import QTimer, Qt, QPoint, QPointF, QSize, QRectF, QObject
+import param
+from matplotlib.backends.backend_agg import FigureCanvasAgg as FigureCanvas
+from matplotlib.figure import Figure
+
+#
+# If is_x, this is viewwidth by projsize, otherwise it is projsize by viewheight!
+#
+class ProjWidget(QWidget):
```

Comment:
Proj = projection, right?

Reply:
Yes.  Originally there was a ProjH and a ProjV, which wasn't horrible when they were just displaying images produced by the sip blob.  Now that they are doing matplotlib stuff, I thought it was worth having some "if X else Y" code inside to make them the same.